frappe/docs/user/en/guides/basics/client-side-api.md

```diff
@@ -0,0 +1,84 @@
+Web applications are based on Client-Server architecture. In Client-Server architecture communication takes place in form of request response calls.
+
+Client initiates the request, server accept the reuest, process it and sends response back to client.
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img//client_server_communication.png">
+
```

Comment:
flow chart probably not required.

Also use serif type fonts while making a flowchart